File: Programmation de Jeux Vidéos III/Semaine 01/Exercises/Cours 02/ExercicesConversionProgramme/FonctionsDiverses/Point.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FonctionsDiverses
{
  class Point
  {
    private int x=0;
    private int y=0;

    public int GetY()
    {
      return y;
    }

    public int GetX()
    {
      return x;
    }

    public Point()
    {
      x = 0;
      y = 0;
    }

    public Point(int newX, int newY)
    {
      x = newX;
      y = newY;
    }

    public float GetDistanceToOrigin()
    {
      return (float)Math.Sqrt(x * x + y * y);
    }

  }
}

File: Programmation de Jeux Vidéos III/Semaine 01/Exercises/Cours 02/ExercicesConversionProgramme/Solution Traduite/Point.cpp
//Les includes:
#include <Math>
//Les includes standards:
#include "point.h";

//Contenu:
class Point
{

private:
    //Variables:
    int x = 0;
    int y = 0;
    //Méthodes: (vide)

public:
    //Variables: (vide)
    int GetX()
    {
        return x;
    }

    int GetY()
    {
        return y;
    }

    Point()
    {
        x = 0;
        y = 0;
    }

    float GetDistanceToOrigin()
    {
        //return (float)
    }
};